tests/database/test_journal_db.py

```diff
@@ -1,14 +1,15 @@
-from uuid import uuid4
-
 from eth_utils import ValidationError
 from hypothesis import (
     given,
+    settings,
     strategies as st,
 )
+from hypothesis.stateful import Bundle, RuleBasedStateMachine, rule
 import pytest
 
 from eth.db.backends.memory import MemoryDB
 from eth.db.journal import JournalDB
+from eth.db.slow_journal import JournalDB as SlowJournalDB
```

Comment:
I see that you are using some hypothesis magic to test the new implementation against the old one but I'm still wondering...what's the intended process here, do we plan to keep `eth.db.slow_journal` as a reference point or was this just something used for debugging that can now be dropped? :thinking:

Reply:
Yeah, I am sort of imagining keeping it around until a little after the release, and then dropping it. Just wanted to make sure there aren't any more hidden bugs in the new one.